packages/host/app/services/matrix-service.ts

```diff
@@ -1470,11 +1491,13 @@ export default class MatrixService extends Service {
           event,
         );
       } else {
+        (event.content as any).origin_server_ts = event.origin_server_ts;
```

Comment:
Is this for ease of debugging, so the `content` passed to subscribers has the timestamp?

Reply:
I think we should probably get rid of this now.